leaflet.photon.js

```diff
@@ -114,7 +115,7 @@ L.PhotonBaseSearch = L.PhotonBase.extend({
     },
 
     createResultsContainer: function () {
-        this.resultsContainer = L.DomUtil.create('ul', 'photon-autocomplete', document.querySelector('body'));
+        this.resultsContainer = L.DomUtil.create('ul', 'photon-autocomplete', document.querySelector(this.options.targetElement));
```

Comment:
The move seems legit, but I'm not sure about the naming: `targetElement`, while the value it's not an element (it's a CSS selector) and `target` is a bit vague imho.
So I wonder if a better option would be to have a `resultsContainer` option, that would be undefined by default, and here we could do something like this:
```javascript
this.resultsContainer = this.options.resultsContainer || L.DomUtil.create('ul', 'photon-autocomplete', document.querySelector('body'));
```

That way users have the full control over the final element used, including which parent, but also the class or even the tag type.
Thoughts?

Reply:
Thank you for your input. You're completely right about the naming. Also your code is a good addition because like you said you'll give the user not only control over the parent selector, but also over the container used for the result. However, when users change the tag-type to something like `<div>` the DOM-structure will go wrong because of the result-tag:

```javascript
var el = L.DomUtil.create('li', '', this.resultsContainer);
```

We can use the same solution as before, but the container always needs to be `this.resultsContainer`.

```javascript
var el = this.options.result || L.DomUtil.create('li', '', this.resultsContainer);
```

the naming is again debatable. Another solution is to give the user only the option to change the tag- and classname.
What are your thoughts about this? :)

Reply:
Erk, sorry, time is running!
I'd say let's keep it simple for now, and only deal with the whole container (the result element is a bit different, given an element is created for each result, vs the container needed once and for all).